tests/test_document.py

```diff
@@ -291,12 +306,12 @@ def body_(self, request):
         return instance_mock(request, _Body)
 
     @pytest.fixture
-    def _block_width_prop_(self, request):
-        return property_mock(request, Document, '_block_width')
```

Comment:
I maintain fixture components in strict alphabetical order by name. This one was out of order, which explains this change not directly related to this test. Keeping them alphabetical speeds finding the one you're after when scanning the source, especially when code-folding and search aren't handy.